levels/sre.tex

```diff
@@ -0,0 +1,368 @@
+    OIDC, and mTLS. Can setup alerting on security systems like CloudWatch.
+
+    \bigbreak
+
+    Understands advanced operating system security concepts. Can utilize
+    Mandatory Access Control (MAC) systems like as SELinux or AppArmor. Understands
+    container security. Can utilize control groups and namespaces to isolate and
+    application.
+    &
+
+    % Systems Engineering
+    &
+
+    % SRE
+    Can use declarative languages like Terraform to build and manage infrastructure.
+
+    \bigbreak
+
+    Demonstrates advanced knowledge of Kubernetes deployments (CRDs, ingress
+    controllers, Load Balancers).
+
+    \bigbreak
+
+    Can build and maintain reliable production CI/CD pipelines for build, test,
+    and release.
+
+    \bigbreak
+
+    Demonstrates advanced knowledge of AWS. May have certification like AWS
+    Certified Solutions Architect.
+    \\ [13em]
+\hline
+    % Level
+    4
+    &
+
+    % Product
+    \cellcolor{blue!25}
+    Leads the implementation of the isolated feature/improvement that
+    measurably and significantly impacts business outcomes from gathering
+    requirements to getting to the market stage.
+    &
+
+    % Communication
+    Writes technical articles/blog posts, delivers tech and lightning talks
+    representing the company's technical vision.
+    &
+
+    % Networking
+    Can write (or heavily modify) network infrastructure tooling (like Cilium)
+    to provide service level load balancing, multi-region connectivity, and
+    observability.
+    &
+
+    % Security
+    Writes technical articles on security aspects of the system, implements
+    significant security product innovations in the area delivered to customers.
+
+    \bigbreak
+
+    Understands and can apply advanced cryptographic principles. Understands
+    hashing (including for anonymization), when to use symmetric and asymmetric
+    cryptography, cipher modes, and TLS versions.
+
+    \bigbreak
+
+    Understands and can apply advanced network security principles. Understands
+    data extrusion prevention. Understands DDoS mitigation. Understands how to
+    monitor systems for rootkits and can deploy mitigation strategies when a system
+    is under attack.
+    &
+
+    % Systems Engineering
+    Can write data-race and deadlock free code. Can implement safe and
+    concurrent and/or parallel systems using minimum amount of shared state,
+    granular locking - systems that are easy to read, extend and troubleshoot.
+    &
+
+    % SRE
+    Can build and operate large scale, stable, and reliable production platform
+    environments like Kubernetes.
+
+    \bigbreak
+
+    Not expected to find any faults or provide any further suggestions in the
+    design document submitted by an Level 4 engineer.
+    \\ [13em]
+\hline
+    % Level
+    5
+    &
+
+    % Product
+    \cellcolor{blue!25}
+    Leads the implementation of a new product line or significant part of the
```

Comment:
Do we realistically expect SREs to work on product features? As in, I'd love to personally, but I have seen this only once on practice.